### PR TITLE
Feature inquiries list -- conditional displays of information in details page

### PR DESCRIPTION
**Pull request by mineralsun**
Made displays of each survey answer (cleaning, moving, etc) conditional so that in the details page of inquiries, the information displayed is tied to whether or not they answered yes to those questions! 

-- Seth or anyone that might need it : feel free to copy some of my conversion functions to display certain info and keep the cards looking clean if you want!

### CODE REVIEW

File: src/redux/sagas/inquiries.saga.js
Comment (mtorres913) [inline src/redux/sagas/inquiries.saga.js:1]: Console log change

File: src/components/Inquiries/Inquiries.jsx
Comment (mtorres913) [inline src/components/Inquiries/Inquiries.jsx:1]: code comment removed

File: src/components/Inquiries/InquiryDetails.jsx
Comment (mtorres913) [inline src/components/Inquiries/InquiryDetails.jsx:1]: Appears to be up to date with existing main. Looks good to me. Recommend another group member review.

**Review by mtorres913 — COMMENTED** [review]
Appears to be up to date with existing main. Looks good to me. Recommend another group member review and approve.

**Review by Svtucci — APPROVED** [review]

**Review by leighstephenson — APPROVED** [review]

**Pull request merged**